src/App.tsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const TOTAL_ITEMS = 42;
+const showItemsPerPage = ['3', '5', '10', '20'];
 
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = useState(showItemsPerPage[1]);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const handlePerPageChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setItemsPerPage(event.target.value);
+    setCurrentPage(1);
+  };
+
+  const start = currentPage * +itemsPerPage - +itemsPerPage + 1;
```

Comment:
if you change the 'showItemsPerPage' array, you won't need to write ' + ' before each variable